docs-src/concepts/what-is-an-update.md

```diff
@@ -0,0 +1,44 @@
+---
+id: what-is-an-update
+title: What is an Update?
+sidebar_label: Update
+description: A Update is a request to a workflow that returns a response.
+tags:
+  - term
+  - updates
+  - explanation
+---
+
+An Update is a request and response from a Temporal Client to a [Workflow Execution](/concepts/what-is-a-workflow-execution).
+
+- [How to develop, send, and handle Updates in Go](/go/updates)
+
+An Update is the full request and response to and from a running Workflow Execution.
+The Workflow must have a function to handle the Update.
+Unlike a [Signal](/concepts/what-is-a-signal) handler, the Update handler function can mutate the state of the Workflow while also returning a value to the caller.
+The Update handler listens for Updates by the Update's name.
+
+When there is the potential for multiple Updates to cause a duplication problem, Temporal recommends adding idempotency logic to your Update handler that checks for duplicates.
+
+An Update has four phases.
+
+1. **_Admission_** - The Temporal Cluster first validates Update submissions against configured resource utilization limits.
```

Comment:
```suggestion
1. **Admission.** The Temporal Cluster first validates Update submissions against configured resource utilization limits.
```
Changed list-item header style throughout per https://developers.google.com/style/lists#description-lists-that-use-run-in-headings.